bin/install-rok8s-requirements

```diff
@@ -70,7 +83,7 @@ if ! hash gcloud 2>/dev/null; then
 fi
 
 # make sure awscli is installed
-if ! hash aws 2>/dev/null && hash pip 2>/dev/null; then
```

Comment:
Should we leave this in, just in case the pip install fails?

Reply:
I almost did that, but I think that extra condition was part of the reason it took us so long to find this. I think I'd rather have it error than fail silently here.